jobs/config.json

```diff
@@ -10904,9 +10904,11 @@
       "--deployment=gke",
       "--env=TAG=5f346f6ec7082ec6517f6ae6e6a4054b68f8f540",
       "--extract=ci/latest-1.9",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--gcp-node-image=gci",
       "--gcp-zone=us-central1-f",
       "--gke-create-command=container clusters create --quiet --num-nodes=4 --enable-kubernetes-alpha --no-enable-legacy-authorization --addons=Istio --istio-config=auth=NONE",
+      "--gke-command-group=alpha",
```

Comment:
It may be better to put alpha in gke-create-command but this will work too.

Reply:
the difference is https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L160-L166, I'm not sure in this case we want or don't want to enforce the api version, or it doesn't matter.